src/main/java/dev/miku/r2dbc/mysql/authentication/MySqlClearAuthProvider.java

```diff
@@ -0,0 +1,42 @@
+package dev.miku.r2dbc.mysql.authentication;
```

Comment:
Please format the code to the style of `intellij-style.xml`, just like other implementations of `MySqlAuthProvider`.